meshio/medit/_medit.py

```diff
@@ -47,7 +48,10 @@ def read_buffer(f):
             break
 
         line = line.strip()
-        if len(line) == 0 or line[0] == "#":
+        if len(line) == 0:
+            continue
+        if line[0] == "#":
```

Comment:
_All_ comments are considered information? Well, I guess one can't say any better.

Reply:
Information is usually in the first line, but can be elsewhere.